packages/shared/lib/contexts/wallet/implicit-account-creation/actions/handleNewOutputForImplicitAccountCreation.ts

```diff
@@ -0,0 +1,25 @@
+import { get } from 'svelte/store'
+import { WalletEvent, NewOutputWalletEvent, WalletEventType } from '@iota/sdk/out/types'
+import { activeWallets } from '@core/profile'
+import { WalletApiEventHandler, loadWallet, validateWalletApiEvent } from '@core/wallet'
+
+export function handleNewOutputForImplicitAccountCreation(walletId: string): WalletApiEventHandler {
+    return (error: Error, rawEvent: WalletEvent) => {
+        validateWalletApiEvent(error, rawEvent, WalletEventType.NewOutput)
+        if (rawEvent.type === WalletEventType.TransactionInclusion) {
+            handleNewOutputForImplicitAccountCreationInternal(walletId, rawEvent as NewOutputWalletEvent)
+        }
+    }
+}
+
+export async function handleNewOutputForImplicitAccountCreationInternal(
```

Comment:
Let's get rid of this, I would move this logic to `handleNewOutputEventInternal.ts`